xgboost_ray/data_sources/modin.py

```diff
@@ -14,7 +14,8 @@
 
 try:
     import modin  # noqa: F401
-    MODIN_INSTALLED = modin.__version__ >= "0.9.0"
+    from distutils.version import LooseVersion
+    MODIN_INSTALLED = LooseVersion(modin.__version__) >= LooseVersion("0.9.0")
```

Comment:
Should we use the more recent [packaging.version.parse](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python) instead?

Reply:
Yeah I didn't want to use that since I believe that's an additional dependency that needs to be installed and would thus have to be packaged with xgboost-ray. `distutils` is a built in package. Also `LooseVersion` is being used in tensorflow (https://github.com/tensorflow/tensorflow/search?q=LooseVersion) so I think should be fine.